index.html

```diff
@@ -27,8 +27,16 @@
               <li class="nav-link">Contact</li>
             </ul>
         </nav>
+        <nav class="portfolioM">
+          <p class="portfolioB"><i class="fa fa-times"></i></p>
+          <ul class="portfoliomenu">
+            <li class="menu"><a href="#hero" id="mbmu">portfolio</a></li>
+            <li class="menu"><a href="#myself" id="mbmu">About</a></li>
+            <li class="menu"><a href="#form" id="mbmu">contact</a></li>
+          </ul>
+          </nav>  
```

Comment:
- According to the project requirements, your mobile version should be responsive for all screen sizes between `375px` and `765px`. Your mobile menu is currently not responsive, please fix this 🙏 so your mobile menu only fits across all mobile screen sizes 👍 
Your project:
![rjdrtjdrt](https://github.com/Danny-imo/my-project/assets/73607512/6f4412ab-10a8-4b3b-8d33-f75612372f26)

- **OPTIONAL** - Please consider making your portfolio generally responsive across all mobile screen sizes i.e `375px` and `765px`. This would ensure your portfolio renders properly on all screen sizes. Please fix this 🙏 

- **OPTIONAL** - You have done well so far 👍, but to help improve the semantics of your page, It is not advisable to have two `nav` tags on your page, I suggest you have just one `nav` tag and style it for both desktop and mobile versions using CSS, please fix this 🙏